prin

```diff
@@ -70,7 +72,7 @@ show_tags () {
   local pr=$1
   local initial_commit=$2
   shift 2
-  local cp_commits=($*)
+  local cp_commits=("$*")
```

Comment:
There is a good chance this should be `"$@"` instead, ref: https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions